amethyst_renderer/src/pass/shaders/vertex/sprite.glsl

```diff
@@ -39,8 +45,8 @@ void main() {
     float tex_v = positions[gl_VertexID][1];
 
     vec2 uv = pos + tex_u * dir_x + tex_v * dir_y;
-    tex_uv = texture_coords(vec2(tex_u, tex_v), u_offset, v_offset);
-
+    vertex.tex_uv = texture_coords(vec2(tex_u, tex_v), u_offset, v_offset);
+    vertex.color = color;
     vec4 vertex = vec4(uv, depth, 1.0);
```

Comment:
does glsl support shadowing?
I thought it would clash

Reply:
This isn't technically shadowing, as `vertex.color` is in a different namespace from `color`.

Reply:
But to your other comment, I tested the output of all these shaders.

Reply:
ah cool